components/hab/src/main.rs

```diff
@@ -287,7 +287,8 @@ fn sub_origin_key_upload(ui: &mut UI, m: &ArgMatches) -> Result<()> {
 
 fn sub_pkg_binlink(ui: &mut UI, m: &ArgMatches) -> Result<()> {
     let ident = PackageIdent::from_str(m.value_of("PKG_IDENT").unwrap())?;
-    let dest_dir = Path::new(m.value_of("DEST_DIR").unwrap_or(DEFAULT_BINLINK_DIR));
+    let env_or_default = henv::var(BINLINK_DIR_ENVVAR).unwrap_or(DEFAULT_BINLINK_DIR.to_string());
```

Comment:
We definitely can. I added that in case it was useful elsewhere. The syntax here was just to match the way we're handling other ENVVAR stuff for the depot bits.